### PR TITLE
Renamed `IOGoogleCloudHttp` to `IOCloudHttpFunction`

### PR DESCRIPTION
**Pull request by Chingles2404**

### CODE REVIEW

**Comment by armanbilge** [comment]
> Would it make sense to rename the GoogleEventFunction as well?

This was committed by mistake 😅 it's not yet part of the build. We'll remove it in a follow-up PR.

File: google-cloud-http4s/jvm/src/main/scala/feral/google-cloud/http4s/IOCloudHttpFunction.scala [hunk 1]
```diff
@@ -37,7 +37,7 @@ import org.typelevel.ci.CIString
 
 import scala.util.control.NonFatal
 
-object IOGoogleCloudHttp {
+object IOCloudHttpFunction {
   def fromHttpRequest(request: HttpRequest): IO[Request[IO]] = for {
```
[the remark this answers is not in the document]
Comment (Chingles2404) [reply google-cloud-http4s/jvm/src/main/scala/feral/google-cloud/http4s/IOCloudHttpFunction.scala:41]: Should the `private[googlecloud]` be added to the `writeResponse` method below as well?

[the remark this answers is not in the document]
Comment (armanbilge) [reply google-cloud-http4s/jvm/src/main/scala/feral/google-cloud/http4s/IOCloudHttpFunction.scala:41]: Oh yes! These methods don't need to be public.

File: google-cloud-http4s/js/src/main/scala/feral/google-cloud/http4s/IOCloudHttpFunction.scala [hunk 2]
```diff
@@ -28,18 +28,18 @@ import org.http4s.nodejs.ServerResponse
 import scala.scalajs.js
 import scala.scalajs.js.annotation._
 
-object IOGoogleCloudHttp {
+object IOCloudHttpFunction {
   @js.native
   @JSImport("@google-cloud/functions-framework", "http")
   def http(
```
Comment (armanbilge) [inline google-cloud-http4s/js/src/main/scala/feral/google-cloud/http4s/IOCloudHttpFunction.scala:34]: Oh, I missed this one. We can make it private too!
```suggestion
  private[googlecloud] def http(
```

**Review by armanbilge — APPROVED** [review]
nice, thanks for tidying this up!

**Pull request merged**